src/main/kotlin/com/terraformation/backend/accelerator/VariableValueStatusNotifier.kt

```diff
@@ -0,0 +1,20 @@
+package com.terraformation.backend.accelerator
+
+import com.terraformation.backend.accelerator.event.VariableValueUpdatedEvent
+import com.terraformation.backend.db.docprod.VariableWorkflowStatus
+import com.terraformation.backend.documentproducer.db.VariableWorkflowStore
+import jakarta.inject.Named
+import org.springframework.context.event.EventListener
+
+@Named
+class VariableValueStatusNotifier(
```

Comment:
Perhaps "updater" (instead of "notifier") would make more sense here. This class does not notify when it observes the event.